module/VuFind/src/VuFind/ChannelProvider/AlphaBrowse.php

```diff
@@ -147,7 +147,7 @@ public function __construct(
      */
     public function setOptions(array $options)
     {
-        $this->channelSize = $options['channelSize'] ?? 20;
+        $this->batchSize = $options['batchSize'] ?? 20;
```

Comment:
The comment says default 24, but it's defaulting to 20. Intentional?


```suggestion
        $this->batchSize = $options['batchSize'] ?? 24;
```